src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid implements Figure {
+    private int top;
+    private int bottom;
+    private int height;
+    private String color;
+
+    public IsoscelesTrapezoid() {
+        this.top = top;
+        this.bottom = bottom;
+        this.height = height;
+        this.color = color;
```

Comment:
The constructor is not receiving any parameters but is trying to assign them. You should define parameters in the constructor signature and pass them when creating an instance of IsoscelesTrapezoid.